rust/arrow/src/util/integration_util.rs

```diff
@@ -521,6 +523,7 @@ fn json_from_col(col: &ArrowJsonColumn, data_type: &DataType) -> Vec<Value> {
                 converted_col.as_slice(),
             )
         }
+        DataType::Null => vec![],
```

Comment:
Shouldn't the null datatype contain N null values where N is the length of the array?